### PR TITLE
Post crud

### PR DESCRIPTION
**Pull request by ThomasG51**

### CODE REVIEW

File: src/Repository/PostRepository.php [hunk 1]
```diff
@@ -49,58 +71,130 @@ public function findAll() : array
...
+            $instance->setUser($this->userManager->findOne($post['email']));
+            $instance->setCategory($this->categoryManager->findOne($post['name']));
+            $instance->setTags($this->tagsLineManager->findTagsByPost($post['slug']));
+
+            return $instance;
+        }
+        else
+        {
+            return false;
+        }
```
Comment (TBoileau) [inline src/Repository/PostRepository.php:109]: Refactoring

File: src/Repository/TagsRepository.php [hunk 2]
```diff
@@ -43,7 +53,16 @@ public function findOne(string $name)
...
+        $tag = $query->fetch();
+
+        if($tag)
+        {
+            return new Tags($tag['id'], $tag['name']);
+        }
+        else
+        {
+            return false;
+        }
```
Comment (TBoileau) [inline src/Repository/TagsRepository.php:65]: Refactoring

**Review by TBoileau — CHANGES_REQUESTED** [review]

File: src/Repository/PostRepository.php (hunk 1 above)
Comment (ThomasG51) [reply src/Repository/PostRepository.php:109]: done

File: src/Repository/TagsRepository.php (hunk 2 above)
Comment (ThomasG51) [reply src/Repository/TagsRepository.php:65]: done

**Review by TBoileau — APPROVED** [review]

**Pull request merged**